.codespellrc

```diff
@@ -0,0 +1,3 @@
+[codespell]
+skip = .git,target,Cargo.toml,Cargo.lock,Cargo-minimal.lock,Cargo-recent.lock
+ignore-words-list = crate,ser
```

Comment:
Does it think crate is not a word?

Reply:
Yeah this was really confusing. But "crate" is not in its dictionary. It tries to auto correct to "create".